Test/TestsDatabase/OrderDatabaseTests.cs

```diff
@@ -199,5 +199,42 @@ public void OrdersCanBeWrittenToDatabaseWithTheory(int value)
             }
         }
 
+        [Theory]
+        [InlineData(1)]
+        [InlineData(2)]
+        [InlineData(3)]
+        [InlineData(4)]
+        [InlineData(5)]
+        [InlineData(6)]
+        public void OrdersCanBeWrittenToDatabaseWithTheoryWithHelper(int value)
+        {
+            using (var contextHelper = new ContextHelper())
+            {
+                var expectedOrders = contextHelper.Context.Orders.ToList();
+
+                expectedOrders.Count().ShouldBe(0);
+
+                contextHelper.Context.Users.Add(CreateValidEntities.User(1));
+                contextHelper.Context.SaveChanges();
+
+                for (int i = 0; i < value; i++)
+                {
+
+                    var order = CreateValidEntities.Order(i + 1);
+                    order.Creator = contextHelper.Context.Users.FirstOrDefault();
+                    contextHelper.Context.Orders.Add(order);
+                }
+
+
+                contextHelper.Context.SaveChanges();
+
+                var updatedOrders = contextHelper.Context.Orders.ToList();

```

Comment:
@jSylvestre would be much more efficient to just do a count() here at the DB level instead of grabbing back the whole list and counting the elements